src/plugins/intel_gpu/src/graph/impls/ocl/kernel_selector_helper.h

```diff
@@ -255,13 +255,15 @@ inline kernel_impl_params canonicalize_fused_shapes(const kernel_impl_params& im
 
     for (auto& fd : updated_impl_params.fused_desc) {
         if (fd.is_type<eltwise>() && fd.total_num_deps == 2) {
-            auto out_pshape = updated_impl_params.output_layouts[0].get_partial_shape();
+            if (updated_impl_params.input_layouts.size() > size_t(fd.outer_dep_start_idx)) {
```

Comment:
Now there is fd.has_outer_dep()

Reply:
Got it! But still line 258 condition is necessary because of line 263.